modules/data/src/main/java/io/fluo/webindex/data/spark/IndexUtil.java

```diff
@@ -145,28 +132,76 @@ private static void addRCV(List<Tuple2<RowColumn, Long>> tuples, String r, Colum
             .getColumn().getFamily().toString().equals(Constants.PAGES)));
 
     JavaPairRDD<RowColumn, Bytes> linkIndex =
-        filteredTopCounts.mapToPair(new PairFunction<Tuple2<RowColumn, Long>, RowColumn, Bytes>() {
-          @Override
-          public Tuple2<RowColumn, Bytes> call(Tuple2<RowColumn, Long> tuple) throws Exception {
-            RowColumn rc = tuple._1();
-            String cf = rc.getColumn().getFamily().toString();
-            String cq = rc.getColumn().getQualifier().toString();
-            byte[] val = tuple._2().toString().getBytes();
-            if (cf.equals(Constants.INLINKS)
-                || (cf.equals(Constants.PAGE) && cq.startsWith(Constants.CUR))) {
-              val = rc.getColumn().getVisibility().toArray();
-            }
-            return new Tuple2<>(new RowColumn(rc.getRow(), new Column(cf, cq)), Bytes.of(val));
+        filteredTopCounts.mapToPair(tuple -> {
+          RowColumn rc = tuple._1();
+          String cf = rc.getColumn().getFamily().toString();
+          String cq = rc.getColumn().getQualifier().toString();
+          byte[] val = tuple._2().toString().getBytes();
+          if (cf.equals(Constants.INLINKS)
+              || (cf.equals(Constants.PAGE) && cq.startsWith(Constants.CUR))) {
+            val = rc.getColumn().getVisibility().toArray();
           }
+          return new Tuple2<>(new RowColumn(rc.getRow(), new Column(cf, cq)), Bytes.of(val));
         });
 
     return linkIndex;
   }
 
-  public static JavaPairRDD<RowColumn, Bytes> filterRank(JavaPairRDD<RowColumn, Bytes> linkIndex) {
-    JavaPairRDD<RowColumn, Bytes> filteredLinkIndex =
-        linkIndex.filter(t -> !t._1().getColumn().getFamily().toString().equals(Constants.RANK));
-    return filteredLinkIndex;
+  public static JavaPairRDD<RowColumn, Bytes> createFluoIndex(
+      JavaPairRDD<RowColumn, Bytes> accumuloIndex) {
+    JavaPairRDD<RowColumn, Bytes> fluoIndex =
+        accumuloIndex
+            .filter(t -> !t._1().getColumn().getFamily().toString().equals(Constants.RANK));
+    return fluoIndex;
+  }
+
+  static class RowColumnComparator implements Comparator<RowColumn>, Serializable {
+    final static RowColumnComparator INSTANCE = new RowColumnComparator();
+
+    @Override
+    public int compare(RowColumn o1, RowColumn o2) {
+      int result;
```

Comment:
Good catch.  The custom comparator is no longer needed now that I made RowColumn comparable in Fluo.  I removed the comparator in de9fe46c12dcc8c0c6